BasePolicies/Governance.md

```diff
@@ -0,0 +1,149 @@
+## Working Group Governance Template
+
+Every Working Group must have a documented governance model that describes the
+basic operation of the Working Group. This must be documented in the repository
+and referenced by the `README.md`.
+
+## Template: *[insert WG name]* Working Group Governance
+
+### Collaborators
+
+The *[insert WG name]* GitHub repository is
+maintained by the WG and additional Collaborators who are added by the
+WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These
+individuals are identified by the WG and their addition as
+Collaborators is discussed during the weekly WG meeting.
+
+_Note:_ If you make a significant contribution and are not considered
+for commit-access log an issue or contact a WG member directly and it
+will be brought up in the next WG meeting.
```

Comment:
This paragraph seems like it could use a bit more polish. Maybe something like this?

> Individuals who have made significant contributions and who wish to be considered for commit-access may create an issue or contact a WG member directly. It is not necessary to wait for a WG member to nominate the individual.